script.js

```diff
@@ -23,34 +27,100 @@ allShows.sort((a, b) => {
   }
   return 0;
 });
-// console.log(allShows);
 
-// ------------------Fetch data from API----------------------------
+// ------------------Fetch data from API--------------------------
 
-function getAllEpisodes(url) {
-  fetch(url)
-    .then((response) => response.json())
-    .then((data) => {
-      allEpisodes = Array.from(data);
-      makePageForEpisodes(data);
-      makeEpisodeMenu(data); //(allEpisodes);
-    })
-    .catch((error) => console.log(error));
+function loadShows(url) {
+  return fetch(url).then((response) => response.json());
 }
 
+//-------------------Setup the page onload------------------------
+
 function setup() {
-  makePageForEpisodes(allEpisodes);
+  hideEpisodeParts();
+  makePageForShows(allShows);
   makeTvShowDropdownMenu();
 }
 
-// -----------------display all episodes-----------------------------
+// -----------------Display all shows-----------------------------
 
-function makePageForEpisodes(episodeList) {
+const imageNotAvailable =
+  "https://sainfoinc.com/wp-content/uploads/2018/02/image-not-available.jpg";
```

Comment:
Try not to use URLs for things like images. If this gets taken down on the other site, you're left with no image. You can save images to your folder structure and use them locally so you know they will always exist :)